<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotControlWidget</class>
 <widget class="QWidget" name="PlotControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="plotTypeGroupBox">
        <property name="title">
         <string>Plot type</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QComboBox" name="plotTypeComboBox">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Collisions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Kinetic energy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Momentum</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Velocity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Count</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="typeSelectionGroupBox">
        <property name="title">
         <string>Shown types</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="selectDiscTypesPushButton">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Select disc types</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="plotSumCheckBox">
        <property name="text">
         <string>Plot sum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Plot interval [ms]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="plotTimeIntervalSpinBox">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
